### PR TITLE
Remove warnings temp

### PR DESCRIPTION
**Pull request by yasuomaidana**
#105 
**Description**

**Related Issue** 
- issue goes here

Fixes # [ISSUE]

- [x] Code
- [ ] User Interface
- [ ] New Feature
- [ ] Documentation
- [ ] Testing

**Code/Quality Assurance Only**

- Bug fix (non-breaking change which fixes an issue)
- This change requires a documentation update (software upgrade on readme file)

**How Has This Been Tested?**

**Additional Info (OPTIONAL)**

**Checklist**

- [ ] My code follows the code style of this project.
- [ ] My UI is responsive
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [x] My changes generate no new warnings
- [ ] The title of my pull request is a short description of the requested changes.

**Screenshots**
   Original                 |   Updated
  :------------------------:|---------------------:
  ** original screenshot ** | ** updated screenshot **

### CODE REVIEW

File: src/Components/Blogs/CurrentBlog.js [hunk 1]
```diff
@@ -50,12 +50,12 @@ const CurrentBlog = (props) => {
     }
     setEditBlog(false);
     setBlogLoading(false);
-  }, []);
+  }, [id]);
```
Comment (atharmohammad) [inline src/Components/Blogs/CurrentBlog.js:53]: please dont remove these warnings due to useEffect they can cause some errors , so make every useeffect warning changes as it is.

File: src/Components/Toolbar/BarItem/Leave.js [hunk 2]
```diff
@@ -10,17 +10,18 @@ import classes from "./tools.module.css";
 function Leave(props) {
   const history = useHistory();
   const location = useLocation();
-  const [room, setRoom] = useState("");
-  const [name, setName] = useState("");
   const [title,setTitle] = useState("");
+  const setRoom = setTitle;
+  const setName = setTitle;
+
```
Comment (atharmohammad) [inline src/Components/Toolbar/BarItem/Leave.js:16]: these changes are not necessary

**Review by atharmohammad — CHANGES_REQUESTED** [review]
After making changes please make sure that project is not giving any errors as there are no errors in the yjs or master branch you pulled from

**Comment by yasuomaidana** [comment]
I have a question, related to errors. I think I'm running the server bad, because even in master in some pages I get errors in console. Is that normal?

File: src/Components/Blogs/CurrentBlog.js (hunk 1 above)
Comment (yasuomaidana) [reply src/Components/Blogs/CurrentBlog.js:53]: I got it

File: src/Components/Toolbar/BarItem/Leave.js (hunk 2 above)
Comment (yasuomaidana) [reply src/Components/Toolbar/BarItem/Leave.js:16]: I'll do it

**Comment by atharmohammad** [comment]
@yasuomaidana  was there were error before you did any changes because warning changes for useeffect which react request many of the times gives error

**Review by yasuomaidana — COMMENTED** [review]
What do you think about this part
image
I also removed all the changes related to useEffect.

File: src/Components/Lockout/FilterContest/FilterContest.js [hunk 3]
```diff
@@ -13,7 +13,7 @@ const FilterContest = (props) => {
     socket.on("Contest-Starting", () => {
       props.setProblemLoading();
     });
-  }, []);
+  }, [props,socket]);
```
Comment (atharmohammad) [inline src/Components/Lockout/FilterContest/FilterContest.js:16]: remove this

**Review by atharmohammad — CHANGES_REQUESTED** [review]
Please do above changes and make sure there is no error when youre running locally

**Comment by atharmohammad** [comment]
> What do you think about this part
> image
> I also removed all the changes related to useEffect.

It's all good

**Comment by yasuomaidana** [comment]
Hi,  I just removed `props,socket` 

Now, talking about this I have some doubts. 
> Please do above changes and make sure there is no error when youre running locally

I don't know if I'm running it well, but even in master branch there are some pages that generates errors. 
## Master branch **NO modifications**
### Pages working well
image
image
image
image
image

### Pages with errors
image
image
image
## Remove warnings temp branch with modifications
### Working well
Same pages
### Errors
image
image
image

## Obtained errors
|:warning: Blogs error|
|:----|
``` console
 Warning: Use the `defaultValue` or `value` props on <select> instead of setting `selected` on <option>.
   at option
    at select
    at div
    at Grid (http://localhost:3000/static/js/vendors~main.chunk.js:13912:35)
    at WithStyles (http://localhost:3000/static/js/vendors~main.chunk.js:42130:31)
   at div
    at BlogPage (http://localhost:3000/static/js/main.chunk.js:21367:88)
    at Route (http://localhost:3000/static/js/vendors~main.chunk.js:160654:29)
    at Switch (http://localhost:3000/static/js/vendors~main.chunk.js:160856:29)
    at CustomRoutes (http://localhost:3000/static/js/main.chunk.js:20639:72)
    at SnackbarProvider (http://localhost:3000/static/js/vendors~main.chunk.js:114309:24)
    at Router (http://localhost:3000/static/js/vendors~main.chunk.js:160289:30)
    at BrowserRouter (http://localhost:3000/static/js/vendors~main.chunk.js:159909:35)
    at App (http://localhost:3000/static/js/main.chunk.js:4677:83)
    at Provider (http://localhost:3000/static/js/vendors~main.chunk.js:155521:20)
````
|:warning: LockOut - Championship errors|
|:----|
|located at `index.js:1 `|
``` console 
Warning: findDOMNode is deprecated in StrictMode. 
findDOMNode was passed an instance of Transition which is inside StrictMode. 
Instead, add a ref directly to the element you want to reference. 
Learn more about using refs safely here: https://reactjs.org/link/strict-mode-find-node
    at div
    at Paper (http://localhost:3000/static/js/vendors~main.chunk.js:21957:23)
    at WithStyles (http://localhost:3000/static/js/vendors~main.chunk.js:42130:31)
    at Alert (http://localhost:3000/static/js/vendors~main.chunk.js:40235:22)
    at WithStyles (http://localhost:3000/static/js/vendors~main.chunk.js:42130:31)
    at Alert
    at Transition (http://localhost:3000/static/js/vendors~main.chunk.js:165359:30)
    at Grow (http://localhost:3000/static/js/vendors~main.chunk.js:14698:24)
    at div
    at ClickAwayListener (http://localhost:3000/static/js/vendors~main.chunk.js:8680:24)
    at Snackbar (http://localhost:3000/static/js/vendors~main.chunk.js:26435:22)
    at WithStyles (http://localhost:3000/static/js/vendors~main.chunk.js:42130:31)
    at Snacker (http://localhost:3000/static/js/main.chunk.js:16708:5)
    at div
    at LockOutPage (http://localhost:3000/static/js/main.chunk.js:22853:24)
    at ConnectFunction (http://localhost:3000/static/js/vendors~main.chunk.js:155808:75)
    at LockoutWrapper (http://localhost:3000/static/js/main.chunk.js:23286:72)
    at Route (http://localhost:3000/static/js/vendors~main.chunk.js:160654:29)
    at Switch (http://localhost:3000/static/js/vendors~main.chunk.js:160856:29)
    at Switch (http://localhost:3000/static/js/vendors~main.chunk.js:160856:29)
    at CustomRoutes (http://localhost:3000/static/js/main.chunk.js:20639:72)
    at SnackbarProvider (http://localhost:3000/static/js/vendors~main.chunk.js:114309:24)
    at Router (http://localhost:3000/static/js/vendors~main.chunk.js:160289:30)
    at BrowserRouter (http://localhost:3000/static/js/vendors~main.chunk.js:159909:35)
    at App (http://localhost:3000/static/js/main.chunk.js:4677:83)
    at Provider (http://localhost:3000/static/js/vendors~main.chunk.js:155521:20)
```
``` console
Warning: Failed prop type: Invalid prop `size` of type `string` supplied to `ReflexElement`, expected `number`.
    at ReflexElement (http://localhost:3000/static/js/vendors~main.chunk.js:158244:5)
    at div
    at ReflexContainer (http://localhost:3000/static/js/vendors~main.chunk.js:157461:5)
    at div
    at LockOutPage (http://localhost:3000/static/js/main.chunk.js:22853:24)
    at ConnectFunction (http://localhost:3000/static/js/vendors~main.chunk.js:155808:75)
    at LockoutWrapper (http://localhost:3000/static/js/main.chunk.js:23286:72)
    at Route (http://localhost:3000/static/js/vendors~main.chunk.js:160654:29)
    at Switch (http://localhost:3000/static/js/vendors~main.chunk.js:160856:29)
    at Switch (http://localhost:3000/static/js/vendors~main.chunk.js:160856:29)
    at CustomRoutes (http://localhost:3000/static/js/main.chunk.js:20639:72)
    at SnackbarProvider (http://localhost:3000/static/js/vendors~main.chunk.js:114309:24)
    at Router (http://localhost:3000/static/js/vendors~main.chunk.js:160289:30)
    at BrowserRouter (http://localhost:3000/static/js/vendors~main.chunk.js:159909:35)
    at App (http://localhost:3000/static/js/main.chunk.js:4677:83)
    at Provider (http://localhost:3000/static/js/vendors~main.chunk.js:155521:20)
```
``` console
Warning: validateDOMNesting(...): <p> cannot appear as a descendant of <p>.
    at p
    at Typography (http://localhost:3000/static/js/vendors~main.chunk.js:33985:28)
    at WithStyles (http://localhost:3000/static/js/vendors~main.chunk.js:42130:31)
    at div
    at Grid (http://localhost:3000/static/js/vendors~main.chunk.js:13912:35)
    at WithStyles (http://localhost:3000/static/js/vendors~main.chunk.js:42130:31)
    at Chat (http://localhost:3000/static/js/main.chunk.js:9138:24)
    at p
    at Typography (http://localhost:3000/static/js/vendors~main.chunk.js:33985:28)
    at WithStyles (http://localhost:3000/static/js/vendors~main.chunk.js:42130:31)
    at div
    at StyledComponent (http://localhost:3000/static/js/vendors~main.chunk.js:41901:28)
    at div
    at TabPanel (http://localhost:3000/static/js/main.chunk.js:9390:5)
    at div
    at ChatPanel (http://localhost:3000/static/js/main.chunk.js:9424:81)
    at div
    at ReflexElement (http://localhost:3000/static/js/vendors~main.chunk.js:158244:5)
    at div
    at ReflexContainer (http://localhost:3000/static/js/vendors~main.chunk.js:157461:5)
    at div
    at LockOutPage (http://localhost:3000/static/js/main.chunk.js:22853:24)
    at ConnectFunction (http://localhost:3000/static/js/vendors~main.chunk.js:155808:75)
    at LockoutWrapper (http://localhost:3000/static/js/main.chunk.js:23286:72)
    at Route (http://localhost:3000/static/js/vendors~main.chunk.js:160654:29)
    at Switch (http://localhost:3000/static/js/vendors~main.chunk.js:160856:29)
    at Switch (http://localhost:3000/static/js/vendors~main.chunk.js:160856:29)
    at CustomRoutes (http://localhost:3000/static/js/main.chunk.js:20639:72)
    at SnackbarProvider (http://localhost:3000/static/js/vendors~main.chunk.js:114309:24)
    at Router (http://localhost:3000/static/js/vendors~main.chunk.js:160289:30)
    at BrowserRouter (http://localhost:3000/static/js/vendors~main.chunk.js:159909:35)
    at App (http://localhost:3000/static/js/main.chunk.js:4677:83)
    at Provider (http://localhost:3000/static/js/vendors~main.chunk.js:155521:20)
```
``` console
Warning: validateDOMNesting(...): <div> cannot appear as a descendant of <p>.
    at div
    at div
    at Panel (http://localhost:3000/static/js/vendors~main.chunk.js:162978:23)
    at div
    at BasicScrollToBottomCore (http://localhost:3000/static/js/vendors~main.chunk.js:161717:23)
    at Composer (http://localhost:3000/static/js/vendors~main.chunk.js:162288:29)
    at BasicScrollToBottom (http://localhost:3000/static/js/vendors~main.chunk.js:161745:29)
    at div
    at Grid (http://localhost:3000/static/js/vendors~main.chunk.js:13912:35)
    at WithStyles (http://localhost:3000/static/js/vendors~main.chunk.js:42130:31)
    at div
    at Grid (http://localhost:3000/static/js/vendors~main.chunk.js:13912:35)
    at WithStyles (http://localhost:3000/static/js/vendors~main.chunk.js:42130:31)
    at Chat (http://localhost:3000/static/js/main.chunk.js:9138:24)
    at p
    at Typography (http://localhost:3000/static/js/vendors~main.chunk.js:33985:28)
    at WithStyles (http://localhost:3000/static/js/vendors~main.chunk.js:42130:31)
    at div
    at StyledComponent (http://localhost:3000/static/js/vendors~main.chunk.js:41901:28)
    at div
    at TabPanel (http://localhost:3000/static/js/main.chunk.js:9390:5)
    at div
    at ChatPanel (http://localhost:3000/static/js/main.chunk.js:9424:81)
    at div
    at ReflexElement (http://localhost:3000/static/js/vendors~main.chunk.js:158244:5)
    at div
    at ReflexContainer (http://localhost:3000/static/js/vendors~main.chunk.js:157461:5)
    at div
    at LockOutPage (http://localhost:3000/static/js/main.chunk.js:22853:24)
    at ConnectFunction (http://localhost:3000/static/js/vendors~main.chunk.js:155808:75)
    at LockoutWrapper (http://localhost:3000/static/js/main.chunk.js:23286:72)
    at Route (http://localhost:3000/static/js/vendors~main.chunk.js:160654:29)
    at Switch (http://localhost:3000/static/js/vendors~main.chunk.js:160856:29)
    at Switch (http://localhost:3000/static/js/vendors~main.chunk.js:160856:29)
    at CustomRoutes (http://localhost:3000/static/js/main.chunk.js:20639:72)
    at SnackbarProvider (http://localhost:3000/static/js/vendors~main.chunk.js:114309:24)
    at Router (http://localhost:3000/static/js/vendors~main.chunk.js:160289:30)
    at BrowserRouter (http://localhost:3000/static/js/vendors~main.chunk.js:159909:35)
    at App (http://localhost:3000/static/js/main.chunk.js:4677:83)
    at Provider (http://localhost:3000/static/js/vendors~main.chunk.js:155521:20)
```
``` console
Warning: validateDOMNesting(...): <form> cannot appear as a descendant of <p>.
    at form
    at div
    at Grid (http://localhost:3000/static/js/vendors~main.chunk.js:13912:35)
    at WithStyles (http://localhost:3000/static/js/vendors~main.chunk.js:42130:31)
    at Chat (http://localhost:3000/static/js/main.chunk.js:9138:24)
    at p
    at Typography (http://localhost:3000/static/js/vendors~main.chunk.js:33985:28)
    at WithStyles (http://localhost:3000/static/js/vendors~main.chunk.js:42130:31)
    at div
    at StyledComponent (http://localhost:3000/static/js/vendors~main.chunk.js:41901:28)
    at div
    at TabPanel (http://localhost:3000/static/js/main.chunk.js:9390:5)
    at div
    at ChatPanel (http://localhost:3000/static/js/main.chunk.js:9424:81)
    at div
    at ReflexElement (http://localhost:3000/static/js/vendors~main.chunk.js:158244:5)
    at div
    at ReflexContainer (http://localhost:3000/static/js/vendors~main.chunk.js:157461:5)
    at div
    at LockOutPage (http://localhost:3000/static/js/main.chunk.js:22853:24)
    at ConnectFunction (http://localhost:3000/static/js/vendors~main.chunk.js:155808:75)
    at LockoutWrapper (http://localhost:3000/static/js/main.chunk.js:23286:72)
    at Route (http://localhost:3000/static/js/vendors~main.chunk.js:160654:29)
    at Switch (http://localhost:3000/static/js/vendors~main.chunk.js:160856:29)
    at Switch (http://localhost:3000/static/js/vendors~main.chunk.js:160856:29)
    at CustomRoutes (http://localhost:3000/static/js/main.chunk.js:20639:72)
    at SnackbarProvider (http://localhost:3000/static/js/vendors~main.chunk.js:114309:24)
    at Router (http://localhost:3000/static/js/vendors~main.chunk.js:160289:30)
    at BrowserRouter (http://localhost:3000/static/js/vendors~main.chunk.js:159909:35)
    at App (http://localhost:3000/static/js/main.chunk.js:4677:83)
    at Provider (http://localhost:3000/static/js/vendors~main.chunk.js:155521:20)
```
|:warning: After creating room errors|
|:----|
|located at `index.js:1 `|
``` console
Warning: findDOMNode is deprecated in StrictMode. 
findDOMNode was passed an instance of Transition which is inside StrictMode. 
Instead, add a ref directly to the element you want to reference. Learn more about using refs safely here:
https://reactjs.org/link/strict-mode-find-node
```
Same as LockOut - Championship but changing 
* LockOutPage for CollabPage
* LockoutWrapper for CollabPageWrapper
* `(http://localhost:3000/static/js/vendors~main.chunk.js:160856:29) at Switch` was  *removed*

``` console
Warning: Failed prop type: Invalid prop `size` of type `string` supplied to `ReflexElement`, expected `number`.
```
Same as LockOut - Championship but changing 
* LockOutPage for CollabPage
* LockoutWrapper for CollabPageWrapper
* `(http://localhost:3000/static/js/vendors~main.chunk.js:160856:29) at Switch` was  *removed*

``` console 
Warning: validateDOMNesting(...): <p> cannot appear as a descendant of <p>.
```
This error appears three times and are same as master branch

``` console
index.js:1 Warning: Failed prop type: The prop `lg` of `Grid` must be used on `item`.
    at WithStyles (http://localhost:3000/static/js/vendors~main.chunk.js:42130:31)
    at Io (http://localhost:3000/static/js/main.chunk.js:11747:72)
    at ConnectFunction (http://localhost:3000/static/js/vendors~main.chunk.js:155808:75)
    at div
    at ReflexElement (http://localhost:3000/static/js/vendors~main.chunk.js:158244:5)
    at div
    at ReflexContainer (http://localhost:3000/static/js/vendors~main.chunk.js:157461:5)
    at div
    at ReflexElement (http://localhost:3000/static/js/vendors~main.chunk.js:158244:5)
    at div
    at ReflexContainer (http://localhost:3000/static/js/vendors~main.chunk.js:157461:5)
    at div
    at CollabPage (http://localhost:3000/static/js/main.chunk.js:21784:24)
    at ConnectFunction (http://localhost:3000/static/js/vendors~main.chunk.js:155808:75)
    at CollabPageWrapper (http://localhost:3000/static/js/main.chunk.js:22182:58)
    at Route (http://localhost:3000/static/js/vendors~main.chunk.js:160654:29)
    at Switch (http://localhost:3000/static/js/vendors~main.chunk.js:160856:29)
    at CustomRoutes (http://localhost:3000/static/js/main.chunk.js:20639:72)
    at SnackbarProvider (http://localhost:3000/static/js/vendors~main.chunk.js:114309:24)
    at Router (http://localhost:3000/static/js/vendors~main.chunk.js:160289:30)
    at BrowserRouter (http://localhost:3000/static/js/vendors~main.chunk.js:159909:35)
    at App (http://localhost:3000/static/js/main.chunk.js:4677:83)
    at Provider (http://localhost:3000/static/js/vendors~main.chunk.js:155521:20)
```

I'm running it in a Mac.

I hope this information is useful.

**Review by atharmohammad — APPROVED** [review]

**Comment by atharmohammad** [comment]
@yasuomaidana  ok so these are warnings these are not errors , so youre running fine , i just requested to remove warnings due ===  and no unique key in map as other warnings has to be taken care with large changes in codebase so dont worry about it, thanks for PR 🎉

**Pull request merged**